distribution/lib/Standard/Visualization/0.0.0-dev/src/Widgets.enso

```diff
@@ -6,6 +6,27 @@ from Standard.Base import all
    Returns full annotation data for all requested arguments.
 get_full_annotations_json : Any -> Text -> Vector Text -> Text
 get_full_annotations_json value call_name argument_names =
+    read_annotation argument =
+        annotation = Meta.get_annotation value call_name argument
+        widget = case annotation of
+            _ : Function -> annotation value
+            _ -> annotation
+
+        js_widget = widget.to_js_object
+        if js_widget.get "values" . is_nothing then js_widget else
+            js_codes = js_widget.get "values" . map (_.get "value")
+            fields = js_widget.field_names
+            values = fields.map f-> if f=="values" then js_codes else js_widget.get f
+            JS_Object.from_pairs (fields.zip values)
+    annotations = argument_names.map (arg -> [arg, read_annotation arg])
+    annotations.to_json
```

Comment:
What is the difference between `get_full_annotations_json` and `get_widget_json`?

The docstring seems the same, but they seem to be doing some different stuff, but I cannot tell what differences are expected here. Would appreciate a short description.